### DISCUSSION TITLE
What will happen to OD in osu!catch in lazer?

### DISCUSSION THREAD

**Discussion by snalgae** [ppy/osu#27633] [Ideas] [answered]
From my limited research I read on wiki that OD in osu!catch affects how high the maximum score can go. 
But because the maximum score on beatmaps has been standardized, I assume OD does nothing in lazer?

**Comment by peppy** [comment, marked as answer]
no idea sorry

**Comment by Joehuu** [comment]
First off, please link the wiki page that says this. I only see https://osu.ppy.sh/wiki/en/Beatmap/Overall_difficulty#osu!catch.

**Reply by snalgae** [reply]
this has the information
https://osu.ppy.sh/wiki/en/Ranking_criteria/osu!catch
added an image too
![Screenshot 2024-03-18 015942](https://github.com/ppy/osu/assets/104220939/ce6fc25a-f133-409c-b3ac-cf8e15991d74)

**Reply by bdach** [reply]
That is partially true because there is a "`difficultyPeppyStars`" quantity that is part of stable score v1 which impacts the "score multiplier" on a given map.

Neither lazer's "classic" scoring, nor "standardised", include OD into the final score. So OD probably does nothing anymore for catch at this point.
